ledger/sandbox-on-x/reference.conf

```diff
@@ -391,6 +434,7 @@ ledger {
         acs-id-page-size = 20000
```

Comment:
@nmarton-da : AFAIR, we dynamically scale that page size based on the number of pages required to for m a stream? Also: do we want to remove the `acs-` prefix as the same kind of params are going to be used for the tx-streams as well?